planetExpress/Game.java

```diff
@@ -0,0 +1,50 @@
+package planetExpress;
+import java.util.Scanner;
+public class Game {
+
+	public static void main(String[] args) 
+	{
+		PlanetExpress planet;
+		Score score;
+		Boolean stable = true;
+		
+		 planet = new PlanetExpress();
```

Comment:
If you are not going to use the undeclared variable anywhere, I would recommend doing the declaration (`PlanetExpress planet`) and the assignment `= new PlanetExpress()` all on one line